benchmarks/asv_delegated_conda.py

```diff
@@ -0,0 +1,198 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
```

Comment:
While it's probably worth noting that this was copied from Iris, the way this is written implies that this is a file that exists within Iris rather than being copied from there.